go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/aws-controllers-k8s/rds-controller
 go 1.14
 
 require (
-	github.com/aws-controllers-k8s/runtime v0.0.5
+	github.com/aws-controllers-k8s/runtime v0.0.6
```

Comment:
@RedbackThomson hey, just FYI, I had issues generating the controller when I updated to runtime and code-generator v0.1.0... I kept getting this error:

```
[jaypipes@thelio code-generator]$ make build-controller SERVICE=rds
Copying common custom resource definitions into rds
Building Kubernetes API objects for rds
Generating deepcopy code for rds
Generating custom resource definitions for rds
Building service controller for rds
Error: template: /home/jaypipes/go/src/github.com/aws-controllers-k8s/code-generator/templates/pkg/resource/resource.go.tpl:67:23: executing "/home/jaypipes/go/src/github.com/aws-controllers-k8s/code-generator/templates/pkg/resource/resource.go.tpl" at <.CRD.SpecIdentifierField>: can't evaluate field SpecIdentifierField in type *model.CRD
make: *** [Makefile:31: build-controller] Error 1
```

checking out code-gen at 0.0.8 and runtime at 0.0.6 solved the above for me. We need to check into this...